h2o-k8s/tests/clustering/h2o-assisted-template.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
So H2O cluster won't use statefull-set anymore?

Reply:
Nope. Won't be necessary, ever. Not necessary for this test. One needs to look into the "nature of this test" a little bit more. I need two H2O pods deployed and cluster them in a very simple and straightforward wy using a python script. StatefulSet could be used as well, but it's too heavy and clever. For this test to `not pass` when required, a more stupid deployment is the perfect fit.